libraries/DataFlash/DataFlash_File.cpp

```diff
@@ -182,6 +183,18 @@ bool DataFlash_File::log_exists(const uint16_t lognum) const
     return ret;
 }
 
+void DataFlash_File::periodic_1Hz(const uint32_t now)
+{
+    if (!io_thread_alive()) {
+        GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "IO thread dead");
```

Comment:
That sounds better. I understand that it is serious, but having a user freak out isn't good. They should remain calm and land it safely, then ask what this message was about.